data/paris-2024.yaml

```diff
@@ -0,0 +1,46 @@
+---
+info:
+  city: Paris
+  link: paris-2024
+  date: 04/23/2024
+  hour: '18:30'
+  icon: icons/paris/paris.svg
+  iconHover: icons/paris/paris-hover.svg
+  hostIcon: icons/paris/leboncoin.png
+  hostName: Lebacoin
+  rsvpsClosed: false
+  food: true
+site:
+  cfp: true
+  city: Paris
+  location: Lebacoin
+  googleMapsLink: https://maps.app.goo.gl/QQGXdYYK1epVQ5ir7
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
It was unexpectedly uneasy to find "office leboncoin" without getting tons of office desk announces

Reply:
really? I just searched it on google maps lol